### PR TITLE
Update docs for SOR using Docker

### PR DESCRIPTION
**Pull request by caike**
## Summary

Add information on how to read and assign signing key and collateral for running the SOR using Docker.

Addresses part of Issue #60 

Please mark the relevant option(s) for your pull request:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring (improving code quality without changing its behavior)
- [x] Documentation update (adding or updating documentation related to the project)

## Checklist

Please ensure that your pull request meets the following criteria:

- [x] I have read the [Contributing Guide](../CONTRIBUTING.md)
- [ ] My code follows the project's coding style and best practices
- [ ] My code is appropriately commented and includes relevant documentation
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests pass
- [x] I have updated the documentation, if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,9 +351,26 @@ the private signing key, the verification key, and the wallet address on the
...
+
+This is the value assigned to `PAYMENT_SIGNING_KEY_CBOR_HEX` when running the SOR via Docker.
+
 It's **recommended** to create and setup a `collateral`. A UTxO with 5 ADAs will
 do the work. But as we mentioned the `collateral` config field is optional.
 
+Use the following command to find and select the UTXO to be used for collateral:
+
+```shell
+cardano-cli query utxo --address $(cat bot.preprod.addr) --testnet-magic 1
```
Comment (Sluder) [inline README.md:368]: Maybe include a note to remove `--testnet-magic 1` for mainnet?

**Review by 4TT1L4 — DISMISSED** [review]
Please add a not about the fact that the `--testnet-magic 1` must be removed for mainnet.

Otherwise it looks good.

File: README.md (hunk 1 above)
Comment (caike) [reply README.md:368]: Done

File: README.md [hunk 2]
```diff
@@ -351,9 +351,28 @@ the private signing key, the verification key, and the wallet address on the
...
```
Comment (sourabhxyz) [inline README.md:368]: Thanks a lot for your contribution @caike ! Maybe worth mentioning that this information can be conveniently obtained from explorer such as `cexplorer.io`.

**Review by sourabhxyz — APPROVED** [review]

**Review by Sluder — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (4TT1L4) [reply README.md:368]: I would still prefer to add a note about mainnet or two commands: one for preprod and one for mainnet.

Comment (caike) [reply README.md]: Updated with these suggestions. However, the section of the README that shows how to create a wallet is using preprod:

```
cardano-cli address build \
    --payment-verification-key-file bot.vkey \
    --testnet-magic 1 \
    --out-file bot.preprod.addr
```

So maybe defaulting to `--mainnet` while using `bot.preprod.addr` as the address might be confusing. Thoughts ?

**Review by adacapo21 — APPROVED** [review]
For me it seems ok after the update.

**Review by sourabhxyz — APPROVED** [review]

**Pull request merged**